`test_storage_s3_queue/test.py::test_replicated` is flaky

[CI DB link](https://play.clickhouse.com/play?user=play#CldJVEgKICAgICd0ZXN0X3N0b3JhZ2VfczNfcXVldWUvdGVzdC5weTo6dGVzdF9yZXBsaWNhdGVkJyBBUyBuYW1lX3N1YnN0ciwKICAgIDE0IEFTIGludGVydmFsX2RheXMKU0VMRUNUCiAgICBjaGVja19zdGFydF90aW1lIEFTIGQsCiAgICBjb3VudCgpLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpCkZST00gY2hlY2tzCldIRVJFICgobm93KCkgLSB0b0ludGVydmFsRGF5KGludGVydmFsX2RheXMpKSA8PSBjaGVja19zdGFydF90aW1lKSBBTkQgKHBvc2l0aW9uKHRlc3RfbmFtZSwgbmFtZV9zdWJzdHIpID4gMCkgQU5EICh0ZXN0X3N0YXR1cyBJTiAoJ0ZBSUwnLCAnRVJST1InLCAnRkxBS1knKSkKR1JPVVAgQlkgZApPUkRFUiBCWSBkIERFU0MK)

https://s3.amazonaws.com/clickhouse-test-reports/PRs/76472/3f393ff3ac8260a6fd798d40889b261c78591437//integration_tests_aarch64_4_4/integration_run_parallel6_0.log

```
        expected_rows = files_to_generate
        for _ in range(100):
            if expected_rows == get_count():
                break
            time.sleep(1)
>       assert expected_rows == get_count()
E       assert 1000 == 800
E        +  where 800 = <function test_replicated.<locals>.get_count at 0xff46305397e0>()

test_storage_s3_queue/test.py:2207: AssertionError
```

--- a/tests/integration/test_storage_s3_queue/test.py
+++ b/tests/integration/test_storage_s3_queue/test.py
@@ -2154,6 +2154,7 @@ def test_replicated(started_cluster):
     node2 = started_cluster.instances["node2"]
 
     table_name = f"test_replicated_{uuid.uuid4().hex[:8]}"
+    mv_name = f"{table_name}_mv"
     dst_table_name = f"{table_name}_dst"
     keeper_path = f"/clickhouse/test_{table_name}"
     files_path = f"{table_name}_data"
@@ -2189,13 +2190,12 @@ def test_replicated(started_cluster):
         started_cluster, files_path, files_to_generate, start_ind=0, row_num=1
     )
 
-    create_mv(node1, f"r.{table_name}", dst_table_name)
-    create_mv(node2, f"r.{table_name}", dst_table_name)
+    create_mv(node1, f"{db_name}.{table_name}", f"{db_name}.{dst_table_name}", mv_name = f"{db_name}.{mv_name}")
 
     def get_count():
         return int(
             node1.query(
-                f"SELECT count() FROM clusterAllReplicas(cluster, default.{dst_table_name})"
+                f"SELECT count() FROM clusterAllReplicas(cluster, {db_name}.{dst_table_name})"
             )
         )
 
@@ -2572,6 +2572,7 @@ def test_registry(started_cluster):
 
     table_name = f"test_registry_{uuid.uuid4().hex[:8]}"
     db_name = f"db_{table_name}"
+    mv_name = f"{table_name}_mv"
     dst_table_name = f"{table_name}_dst"
     keeper_path = f"/clickhouse/test_{table_name}"
     files_path = f"{table_name}_data"
@@ -2614,13 +2615,12 @@ def test_registry(started_cluster):
         started_cluster, files_path, files_to_generate, start_ind=0, row_num=1
     )
 
-    create_mv(node1, f"{db_name}.{table_name}", dst_table_name)
-    create_mv(node2, f"{db_name}.{table_name}", dst_table_name)
+    create_mv(node1, f"{db_name}.{table_name}", f"{db_name}.{dst_table_name}", mv_name = f"{db_name}.{mv_name}")
 
     def get_count():
         return int(
             node1.query(
-                f"SELECT count() FROM clusterAllReplicas(cluster, default.{dst_table_name})"
+                f"SELECT count() FROM clusterAllReplicas(cluster, {db_name}.{dst_table_name})"
             )
         )
 

--- a/tests/integration/test_storage_s3_queue/test.py
+++ b/tests/integration/test_storage_s3_queue/test.py
@@ -2155,19 +2155,20 @@ def test_replicated(started_cluster):
 
     table_name = f"test_replicated_{uuid.uuid4().hex[:8]}"
     mv_name = f"{table_name}_mv"
+    db_name = f"r"
     dst_table_name = f"{table_name}_dst"
     keeper_path = f"/clickhouse/test_{table_name}"
     files_path = f"{table_name}_data"
     files_to_generate = 1000
 
-    node1.query("DROP DATABASE IF EXISTS r")
-    node2.query("DROP DATABASE IF EXISTS r")
+    node1.query(f"DROP DATABASE IF EXISTS {db_name}")
+    node2.query(f"DROP DATABASE IF EXISTS {db_name}")
 
     node1.query(
-        "CREATE DATABASE r ENGINE=Replicated('/clickhouse/databases/replicateddb', 'shard1', 'node1')"
+        f"CREATE DATABASE {db_name} ENGINE=Replicated('/clickhouse/databases/replicateddb', 'shard1', 'node1')"
     )
     node2.query(
-        "CREATE DATABASE r ENGINE=Replicated('/clickhouse/databases/replicateddb', 'shard1', 'node2')"
+        f"CREATE DATABASE {db_name} ENGINE=Replicated('/clickhouse/databases/replicateddb', 'shard1', 'node2')"
     )
 
     create_table(

